package-lock.json

```diff
@@ -0,0 +1,10152 @@
+{
```

Comment:
Why is there a package-lock.json and yarn.lock file?

Reply:
Yeah, the same here, tho. I kinda followed up the template you guys had there, and it also slipped in and didn't get notice by me, lol. I only used the yarn for packages set up.